jobs/candlepinPerformanceTestMaintenance.groovy

```diff
@@ -0,0 +1,38 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+
+job("$baseFolder/CandlepinPerformanceMaintanance") {
```

Comment:
Let's be more descriptive. Let's name it "CandlepinUpdatePerformanceBaseline".

Reply:
Sounds good. I also renamed this new file.